tests/console/sysstat.pm

```diff
@@ -42,17 +47,21 @@ sub run {
     validate_script_output "mpstat -P ALL 2 5 |grep all |wc -l", sub { /6/ };
 
     #header integrity checks:
-    validate_script_output "pidstat",    sub { /UID       PID    %usr %system  %guest   %wait    %CPU   CPU  Command/ };
+    if (is_sle('=12-SP1') || is_sle('=12-SP2') || is_sle('=12-SP4')) {
```

Comment:
Could you please at least put a comment explaining why SP1, SP2, SP4 and not SP3? Also, can we expect same behavior in SP5? So we don't have to change code once there is a new service pack. WDYT?

Reply:
Indeed, will dig deeper in that as it maybe some package version inconsistency on the repos.

Reply:
ok, so there is one update for 12.4 in staging status right now, once it is released sysstat will be the same version on sle12.4, sle12.3 and sle15 and the header should match again for this 3, I will update the code accordingly.

I will add a comment to the code once the update for 12.4 is released, the  pidstat command has different output depending on its version: 
sle12.1 and sle12.2 have sysstat version 10.2 
sle12.3, sle12.4 and sle15 have sysstat version 12.0